integrationtests.py

```diff
@@ -151,6 +151,16 @@ def testRecompileObjectSetOtherDir(self):
         subprocess.check_call(cmd) # Compile once
         subprocess.check_call(cmd) # Compile again
 
+    def testPipedOutput(self):
+        with cd(ASSETS_DIR):
+            cmd = [PYTHON_BINARY, CLCACHE_SCRIPT, "/nologo", "/c", "fibonacci.c", "/Fofibonacci_c.obj"]
```

Comment:
Could you the deleting the cache including the manifests directory? Who knows what is in there ...